connectors/spark/src/main/scala/io/unitycatalog/spark/Constants.java

```diff
@@ -0,0 +1,20 @@
+package io.unitycatalog.spark;
+
+public class Constants {
+    public Constants() {}
+
+    public static final String UNITY_CATALOG_URI = "fs.unitycatalog.uri";
```

Comment:
nit: lets shorten these constant variable by 
- either `UNITY_CATALOG_` --> `UC_`
- or ... we name the class to UCConstants and remove the UC prefix completely. everything will use UCConstants.URI